tests/link/test_jax.py

```diff
@@ -763,18 +754,50 @@ def test_jax_IncSubtensor():
     out_fg = FunctionGraph([], [out_at])
     compare_jax_and_py(out_fg, [])
 
-    st_at = at.as_tensor_variable(x_np[[0, 2], 0, :3])
-    out_at = at_subtensor.inc_subtensor(x_at[[0, 2], 0, :3], st_at)
+    # "Increment" boolean indices
+    mask_at = at.constant(x_np > 0)
+    out_at = at_subtensor.set_subtensor(x_at[mask_at], 1.0)
     assert isinstance(out_at.owner.op, at_subtensor.AdvancedIncSubtensor)
     out_fg = FunctionGraph([], [out_at])
     compare_jax_and_py(out_fg, [])
 
-    # "Increment" boolean indices
+
+def test_jax_IncSubtensors_unsupported():
```

Comment:
This should have an `xfail` mark to be obvious that it's a failure and not desired behavior.